libs/datasets/dataset_utils.py

```diff
@@ -195,6 +195,13 @@ def build_fips_data_frame():
 
 
 def add_county_using_fips(data, fips_data):
+    is_county = data.aggregate_level == AggregationLevel.COUNTY.value
+    # Only want to add county names to county level data, so we'll slice out the county
+    # data and combine it back at the end.
+    not_county = data[~is_county]
+    data = data[is_county]
+    fips_data = fips_data[fips_data.aggregate_level == AggregationLevel.COUNTY.value]
```

Comment:
i mean kinda wish we had a different dataset for state fips and county fips?

Reply:
Yeah, I do wonder if we should just be separating out county/state/country data in the data source itself